tools/src/h5diff/ph5diff_main.c

```diff
@@ -312,5 +312,5 @@ h5diff_exit(int status)
     /* Always exit(0), since MPI implementations do weird stuff when they
      *  receive a non-zero exit value. - QAK
      */
-    HDexit(0);
+    HDexit(status);
```

Comment:
This shouldn't be changes (per my comment immediately above :-)

Reply:
Interesting if there are still MPI implementations that behave weird. On the other hand, just a few lines up we always set status to EXIT_SUCCESS when in parallel, and EXIT_SUCCESS should be equal to 0, so (I think?) this shouldn't be a problem.

Reply:
Worth a try :-)

Reply:
EXIT_SUCCESS is used instead of 0/1 because the "good" and "bad" error codes are platform-specific. VMS/OpenVMS were weird about return codes, for example.